server/testbed/Makefile

```diff
@@ -0,0 +1,41 @@
+# SPDX-FileCopyrightText: 2022 ANSSI
+# SPDX-License-Identifier: Apache-2.0
+#
+hostbus := $(shell lsusb | grep Bluetooth | sed -r 's/([^0-9]*([0-9]*)){1}.*/\2/')
```

Comment:
Maybe more legible if we use the fact that there is Bus and Device in the output, something like:
`s/Bus ([0-9]+)/\1/`
and likewise with Device for hostaddr.

Reply:
ACK